src/structures/Guild.js

```diff
@@ -435,6 +396,7 @@ class Guild extends BaseGuild {
         emojis: data.emojis,
       });
     }
+    super._patch(data);
```

Comment:
Is there a reason this is at the end? If not, this usually goes at the top.

Reply:
noted and addressed in https://github.com/discordjs/discord.js/pull/5490/commits/264b010351d4ca77275b60badd1c681d78f9fee5